src/orquestra/integrations/cirq/conversions/_circuit_conversions.py

```diff
@@ -230,6 +244,11 @@ def _export_to_cirq(obj):
 
     Exporting of user-defined gates is atm not supported.
     """
+    if isinstance(obj, _wavefunction_operations.ResetOperation):
```

Comment:
Instead of adding an `if` statement to check for a reset operation, can we use the `register` method provided by `@singledispatch`? That would be more consistent with how other types of operations are handled. Something like:
```python
@_export_to_cirq.register
def _export_reset_operation_to_cirq(op: _wavefunction_operations.ResetOperation):
    return cirq.ResetChannel()(*map(cirq.LineQubit, op.qubit_indices))
```